lib/fileutils.rb

```diff
@@ -631,16 +631,52 @@ def link_entry(src, dest, dereference_root = false, remove_destination = false)
   end
   module_function :link_entry
 
+  # Copies files from +src+ to +dest+;
+  # +src+ may not be the path to a directory
+  # and +dest+ may not be the path to a file.
   #
-  # Copies a file content +src+ to +dest+.  If +dest+ is a directory,
-  # copies +src+ to +dest/src+.
+  # If +src+ is the path to a file and +dest+ does not exist, copies +src+ to +dest+:
```

Comment:
This is also true if dest is already a file.

Reply:
Fixed.